inspire/modules/authors/testsuite/test_author_record.py

```diff
@@ -0,0 +1,214 @@
+# -*- coding: utf-8 -*-
+## This file is part of INSPIRE.
+## Copyright (C) 2014, 2015 CERN.
+##
+## Invenio is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## Invenio is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Tests for author records."""
+
+from invenio.testsuite import make_test_suite, run_test_suite, InvenioTestCase
+
+
+class AuthorRecordTestCase(InvenioTestCase):
+
+    """Base class for author_base records test."""
+
+    def setUp(self):
+        """Preparing evironment for tests."""
+        self.import_context()
+
+    def import_context(self):
+        """Import the Invenio classes here.
+
+        Makes sure the testing context is valid.
+        """
+        from invenio.modules.records.api import Record
+        self.Record = Record
+
+    def create_record_from_json(self, json_dict):
+        """Return record created from json dictionary.
+
+        :param json_dict: The dictionary in JSON from with author content
+        :return: an author base record
+        """
+        return self.Record.create(json_dict, 'json', model='author')
+
+    def create_record_from_marc(self, xml_string):
+        """Return record created from MARCxml string.
+
+        :param xml_string: The string representing MARCxml of an author
+        :return: an author base record
+        """
+        return self.Record.create(xml_string, 'marc', model='author')
+
+
+class AuthorRecordRequiredFieldsTestCase(AuthorRecordTestCase):
+
+    """Tests for required fields from author_base records."""
+
+    def test_marc_correct_date_split(self):
+        """Here the date split should produce two yearly integers."""
+        xml_string = '<collection><record><datafield tag="035" ind1="" ind2' \
+            '=""><subfield code="9">authorid</subfield><subfield code="a">' \
+            '0000-0002-0003-2345</subfield></datafield><datafield tag="900" ind1' \
+            '="" ind2=""><subfield code="a">9</subfield>' \
+            '</datafield><datafield tag="100" ind1' \
+            '="" ind2=""><subfield code="a">van der Nvenio, I.</subfield>' \
+            '<subfield code="d">1962-1999</subfield></datafield>' \
+            '</record></collection>'
+
```

Comment:
A minor suggestion: 
You could use the parenthesis syntax for multiple line strings. It is considered to be more clear.